contracts/test/proposals/Addresses.sol

```diff
@@ -0,0 +1,173 @@
+pragma solidity =0.8.13;
+
+contract Addresses {
+    mapping(string => address) public mainnet;
```

Comment:
this should be private to keep proper encapsulation and ensure the `addMainnet` function is used.

Reply:
? the way the addresses are fetched is by using the getter of this public mapping (`addresses.mainnet("key") -> address`) so we should keep it public, and there is no way for an external contract to write in this mapping other than using the `addMainnet` function already

Reply:
we could just add a `mainnet()` function that returns the queried item from the mapping